src/main/java/seedu/task/model/RecurringTaskList.java

```diff
@@ -0,0 +1,194 @@
+//package seedu.task.model;
```

Comment:
Could you add this to an unused folder? That way your code can still be collated but we can keep commented code out of the main code base.